diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0f738d1..e388d88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,11 @@ string(COMPARE EQUAL ${CMAKE_SYSTEM_NAME} "Linux" IS_LINUX)
 
 # Project options
 include(CMakeDependentOption)
-cmake_dependent_option(PLUME_SDL_VULKAN_ENABLED "Enable SDL Vulkan integration" OFF IS_LINUX OFF)
+if(ANDROID)
+    set(PLUME_SDL_VULKAN_ENABLED ON)
+else()
+    cmake_dependent_option(PLUME_SDL_VULKAN_ENABLED "Enable SDL Vulkan integration" OFF IS_LINUX OFF)
+endif()
 cmake_dependent_option(PLUME_D3D12_AGILITY_SDK_ENABLED "Enable D3D12 Agility SDK" OFF WIN32 OFF)
 cmake_dependent_option(PLUME_APPLE_RETINA_ENABLED "Enable Apple Retina display support" ON APPLE ON)
 option(PLUME_BUILD_EXAMPLES "Build example applications" OFF)
diff --git a/plume_render_interface_types.h b/plume_render_interface_types.h
index 8be5b05..8b9b91a 100644
--- a/plume_render_interface_types.h
+++ b/plume_render_interface_types.h
@@ -37,10 +37,10 @@ namespace plume {
 #if defined(_WIN64)
     // Native HWND handle to the target window.
     typedef HWND RenderWindow;
-#elif defined(__ANDROID__)
-    typedef ANativeWindow* RenderWindow;
 #elif defined(PLUME_SDL_VULKAN_ENABLED)
     typedef SDL_Window *RenderWindow;
+#elif defined(__ANDROID__)
+    typedef ANativeWindow* RenderWindow;
 #elif defined(__linux__)
     struct RenderWindow {
         Display* display;
